--- haproxy-2.4.7/src/ssl_sock.c~	2021-10-04 14:56:33.000000000 +0200
+++ haproxy-2.4.7/src/ssl_sock.c	2021-10-25 23:49:26.805909724 +0200
@@ -2270,13 +2270,13 @@
 		: SSL_set_min_proto_version(ssl, TLS1_2_VERSION);
 }
 static void ctx_set_TLSv13_func(SSL_CTX *ctx, set_context_func c) {
-#if SSL_OP_NO_TLSv1_3
+#if (HA_OPENSSL_VERSION_NUMBER >= 0x10101000L)
 	c == SET_MAX ? SSL_CTX_set_max_proto_version(ctx, TLS1_3_VERSION)
 		: SSL_CTX_set_min_proto_version(ctx, TLS1_3_VERSION);
 #endif
 }
 static void ssl_set_TLSv13_func(SSL *ssl, set_context_func c) {
-#if SSL_OP_NO_TLSv1_3
+#if (HA_OPENSSL_VERSION_NUMBER >= 0x10101000L)
 	c == SET_MAX ? SSL_set_max_proto_version(ssl, TLS1_3_VERSION)
 		: SSL_set_min_proto_version(ssl, TLS1_3_VERSION);
 #endif
